### PR TITLE
GRW-1014 / Feature / Add SWEDISH_CAR as possible QuoteType

### PR DESCRIPTION
**Pull request by simonauner**
## What?

Adds the possibility to create a quote for `SWEDISH_CAR`

image

image

## Why?

It will be time-consuming to always have to go through the Embark flow to get a quote to be able to work with the updated offer page.

**Ticket(s): [GRW-1014]**

[GRW-1014]: https://hedvig.atlassian.net/browse/GRW-1014?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by simonauner** [comment]
This seems to break creating any other quote though, since Formik cannot handle changing `initialValues` (is my working hypothesis at least)

image

**Comment by simonauner** [comment]
Ok, solved ☝️  by adding `enableReinitialize` to `<Formik>`

File: src/client/pages/OfferDebugger/components/QuoteData.tsx [hunk 1]
```diff
@@ -197,12 +212,29 @@ const getSwedishHouseQuoteValues = (values: any) => {
...
+  }))
+}
+
 const ButtonLoadingIndicator = styled(LoadingDots)`
   display: inline-flex;
   margin-left: 0.5rem;
 `
 
-export const QuoteData: React.FC<OfferProps> = ({ quoteCartId }) => {
+export const QuoteData = ({ quoteCartId }: OfferProps) => {
```
Comment (robinandeer) [inline src/client/pages/OfferDebugger/components/QuoteData.tsx:237]: 🧹

File: src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx [hunk 2]
```diff
@@ -3,6 +3,24 @@ import { ApartmentType } from 'data/graphql'
...
+  email: 'sven.svensson@hedvig.com',
+  data: {
+    street: 'Storgatan 1',
+    zipCode: '12345',
+    livingSpace: 23,
+    numberCoInsured: 1,
+    subType: ApartmentType.Rent,
+  },
+}
+
```
Comment (robinandeer) [inline src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx:23]: Nice that you added this!

Some things are missing that should be required:

- ancillaryArea
- yearOfConstruction
- numberOfBathrooms

`subType` is not needed for house so maybe remove that

**Review by robinandeer — DISMISSED** [review]
Really great :D

File: src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx (hunk 2 above)
Comment (robinandeer) [reply src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx:23]: I think it's fine to leave them out -- you can always change that in the edit modal

Comment (simonauner) [reply src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx]: Ok, removed `subType` from house. The other fields you mentioned are set hard-coded when submitting the form 🤷

**Review by robinandeer — APPROVED** [review]

**Pull request merged**